packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -142,11 +142,25 @@ class Directionality extends InheritedWidget {
 /// This is more efficient than adding and removing the child widget from the
 /// tree on demand.
 ///
+/// ## Opacity Animation
+///
+/// `Opacity` animations should be built using [AnimatedOpacity] rather than
+/// manually updating the `Opacity` Widget.
+///
+/// The 'Opacity' Widget rebuilds its subtree every frame of an animation.
+/// [AnimatedOpacity] uses a [RenderAnimatedOpacity] internally which listens
+/// directly to an `Animation` so that the Widget tree below the
+/// `AnimatedOpacity` is not rebuilt every frame.
+///
 /// See also:
 ///
 ///  * [ShaderMask], which can apply more elaborate effects to its child.
 ///  * [Transform], which applies an arbitrary transform to its child widget at
 ///    paint time.
+///  * [AnimatedOpacity], which efficiently animates opacity at the Widget
+///    level.
+///  * [RenderAnimatedOpacity], which efficiently animates opacity at the render
```

Comment:
no need to point to the render object from this layer

Reply:
@Hixie  #15464 explicitly says to reference RenderAnimatedOpacity, are you sure you want it removed from "See also"?  And do you also want it removed from the earlier paragraph that mentions it?

Reply:
We can chat with @xster to see if he had a particular reason for suggesting that, but the general rule is that in the widgets layer we just talk widgets and pretend the render objects don't exist, and in the render object layer we talk about both (focusing mainly on the render objects but always giving readers links back to the widgets in case they accidentally get lost in the docs).

Reply:
Ah, sorry I just meant refer to widgets that uses RenderAnimatedOpacity (like you're doing above), not RenderAnimatedOpacity directly.